src/storage/prisma.ts

```diff
@@ -185,35 +165,77 @@ export class PrismaStorage implements IDriveStorage {
         operations: Operation[],
         header: DocumentHeader
     ): Promise<void> {
-        try {
-            await tx.operation.createMany({
-                data: operations.map(op => ({
-                    driveId: drive,
-                    documentId: id,
-                    hash: op.hash,
-                    index: op.index,
-                    input: JSON.stringify(op.input),
-                    timestamp: op.timestamp,
-                    type: op.type,
-                    scope: op.scope,
-                    branch: 'main',
-                    opId: op.id,
-                    skip: op.skip,
-                    context: op.context,
-                    resultingState: op.resultingState
-                        ? Buffer.from(JSON.stringify(op.resultingState))
-                        : undefined
-                }))
+        // TODO all operations should belong to the same sync unit
```

Comment:
When we rework the storage API, when adding operations, it should be enforced that all ops belong to the same sync unit.

Reply:
For now they are grouped and iterated by sync unit.